test/instrumentation/modules/@aws-sdk/fixtures/use-client-s3.js

```diff
@@ -129,14 +152,30 @@ async function useClientS3 (s3Client, bucketName) {
   // https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/classes/getobjectcommand.html
   command = new GetObjectCommand({ Bucket: bucketName, Key: key })
   data = await s3Client.send(command)
-  log.info({ data }, 'getObject')
+  // `data.Body` is a *stream*, so we cannot just log it
+  // https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/getobjectcommandoutput.html#body
+  const body = await slurpStream(data.Body)
+  assert(body === content)
+  delete data.Body
+  log.info({ data, body }, 'getObject')
```

Comment:
Reviewer note: This change is unrelated to the bug. With aws-sdk **v3** that `data.Body` is a Node.js *stream* object. Logging it results in an unhelpful and huge log record. This changes to fully read that stream so we can log the S3 object content instead.